### ISSUE TITLE
[ComplexImageView] setVisualizationMode should accept string

### ISSUE THREAD

**Issue by vasole** [silx-kit/silx#2420]
Only an object is accepted now what makes things cumbersome. One has to call getSupportedVisualizationModes and select the appropriate index (4 in case of willing 'Amplitude and Phase' instead of just passing the string)

**Comment by t20100** [comment]
Looking at it, the `Mode` are available in the `ComplexImageView` class, so there is no need to import them from the complex image item module.

Sample code:
```
from silx.gui.plot.ComplexImageView import ComplexImageView

widget = ComplexImageView()
widget.setVisualizationMode(widget.Mode.AMPLITUDE_PHASE)
# or
widget.setVisualizationMode(ComplexImageView.Mode.AMPLITUDE_PHASE)
widget.show()
```

We can update the documentation of `setVisualizationMode` with sample code, but I would leave it as it is.

**Comment by vasole** [comment]
OK. Waiting for the tests to finish.

**Comment by t20100** [comment]
closed by #2423
